app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/column.tsx

```diff
@@ -216,10 +191,13 @@ function SharedWithColumn(props: AssetColumnProps) {
             }}
         >
             {(item.permissions ?? []).map(user => (
-                <UserPermissionDisplay key={user.user.user_email} user={user} />
+                <PermissionDisplay key={user.user.pk} action={user.permission}>
```

Comment:
Oh no we return `pk` from the database? That's a shame. `pk` is meant to be an internal-only field name that we replace with a more friendly identifier describing what this type is. Oh well, suppose it's okay for now.